salt/utils/schedule.py

```diff
@@ -615,6 +620,34 @@ def job_status(self, name):
         schedule = self._get_schedule()
         return schedule.get(name, {})
 
+    def job_running(self, data):
```

Comment:
What is this actually used for?

Reply:
This seems to be a copy of `_check_max_running` - why don't we call that and then return True/False based on the result?

Reply:
They are not identical, the code is similar but `job_running` takes different arguments than `_check_maxrunning` and they have different return values.  We're also setting various things in `_check_maxrunning` in data that is used later, that is not being set in `job_running`.  That being said I can't find where `job_running` is being used and I suspect it might be something that the Enterprise team might have needed, @pizzapanther can you comment if you're using `job_running` for anything?

Reply:
we do not use `job_running` because we can't exclusively rely on one master to check for running schedules, so we have other measures.

I would suggest resurrecting: https://github.com/saltstack/salt/pull/49166

Then you would have tests on how we use the scheduler which would catch any API breakage we our relying on.

Reply:
@dhiltonp Do you have any further comments here?

Reply:
I haven't found any references to 'job_running' in the enterprise code, maybe we should pull this out.

Reply:
There aren't tests for this, either.